src/modules/transaction/components/Multisignature/Multisignature.js

```diff
@@ -87,7 +87,12 @@ const Multisignature = ({
 
   const onDownload = () => {
     const transaction = JSON.parse(transactionToJSON(transactions.signedTransaction));
-    downloadJSON(transaction, `tx-${transaction.id}`);
+    const fileSuffix =
+      transaction?.id ??
```

Comment:
since we have this condition here wouldnt this allow us to download a broken file with `tx-undefined` as the name?

So doing something like this might make sense?
```
if(transaction?.id) {
  const fileSuffix = `registerMultisig-${cryptography.address.getLisk32AddressFromPublicKey(
        transactions.signedTransaction.senderPublicKey
      )}`;
  downloadJSON(transaction, `tx-${fileSuffix}`);
} else {
  // show error message, "Transaction id is missing" or something?
}
```

Reply:
@oskarleonard Transaction ID would exist if the transaction has been signed by any of the parties involved. However, the ID would not exist when a multisig registration is carried out but none of the transaction members has signed. So based on the condition, `tx-undefined` can not be a possible name because the second condition is used if ID is undefined